src/components/atoms/typography/ms-typography.styles.ts

```diff
@@ -1,43 +1,31 @@
-import { css } from 'lit'
-
-import { setFont } from '~/utils'
-
-//@TODO: modify styles with var on style-dictionary & theme
+import { css, unsafeCSS } from 'lit'
+
+import { typographies, type MsTypographies } from '~/styles/tokens/default/typographies'
+import { defaultMSTheme } from '~/theme'
+import { getThemeColor } from '~/utils'
+
+const typographyStylesGenerator = () => {
+  const typographyVariants = Object.keys(typographies)
+
+  return typographyVariants
+    .map((variant) => {
+      const themeBase = defaultMSTheme.typographies[variant as keyof MsTypographies]
+      return `.${variant} {
+      font-size: var(--${variant}-font-size, ${themeBase.fontSize}px);
+      font-family: var(--${variant}-font-family, ${themeBase.fontFamily});
+      font-weight: var(--${variant}-font-weight, ${themeBase.fontWeight});
+      line-height: var(--${variant}-line-height, ${themeBase.fontSize}px);
+      letter-spacing: var(--${variant}-letter-spacing, ${themeBase.letterSpacing});
+    }`
+    })
+    .join('')
+}
 
 export const msTypographyStyles = css`
-  h1 {
-    ${setFont(56, 64)}
-    font-weight: inherit
-  }
-
-  h2 {
-    ${setFont(48, 56)}
-    font-weight: inherit
-  }
-
-  h3 {
-    ${setFont(40, 48)}
-    font-weight: inherit
-  }
-
-  h4 {
-    ${setFont(32, 40)}
-    font-weight: inherit
-  }
-
-  h5 {
-    ${setFont(28, 32)}
-    font-weight: inherit
-  }
-
-  h6 {
-    ${setFont(24, 28)}
-    font-weight: inherit
-  }
-
-  p {
-    ${setFont(17, 24)}
-    font-weight: inherit;
+  :host > *:first-child {
     margin: 0;
+    box-sizing: border-box;
+    color: ${getThemeColor('gray-900')};
   }
+  ${unsafeCSS(typographyStylesGenerator())}
 `
```

Comment:
Just to understand, now we'll call this method every time we need to use the typographies' styles?

Reply:
Nop, if you want to use typographies, u should use typography component `<ms-typography>`
eg:
```
<button><ms-typography variant="heading-2">Boton</ms-typography></button>
```
and also if you want to use the values of css variables, there are css variables to get those values => `--heading-medium-h1-font-size`